swagger.yaml

```diff
@@ -835,26 +1004,47 @@ components:
         message:
           type: string
           description: Message attached to request
-        projectManagerStatus:
-          $ref: "#/components/schemas/VacationRequestStatus"
-        hrManagerStatus:
-          $ref: "#/components/schemas/VacationRequestStatus"
         createdAt:
           type: string
           format: date-time
           description: Date time of vacation request registration
-        createdBy:
-          type: string
-          format: uuid
-          description: ID of person who modified request last
         updatedAt:
           type: string
           format: date-time
           description: Date time of vacation request update
-        lastUpdatedBy:
+
+    VacationRequestStatus:
+      type: object
+      description: singular vacation request status chance
+      required:
+        - vacationRequestId
+        - person
+        - status
+        - updatedAt
+      properties:
+        id:
           type: string
           format: uuid
-          description: ID of person who modified request last
+          description: ID of vacation request status in UUID
+          default: null
+        vacationRequestId:
+          type: string
+          format: uuid
+          description: ID of vacation request in UUID
+          default: null
+        person:
+          type: integer
+          format: int32
+          description: Person ID in Forecast
+        status:
```

Comment:
Would this be better as statuses?

Reply:
I think it is better as it is sense it is just one status at a time. The enum contains all statuses so I think the name fits it.